### ISSUE TITLE
Cleanup `if` repo post-plugin migration

### ISSUE THREAD

**Issue by jmcook1186** [Green-Software-Foundation/if#729]
**What**
We moved our `if-plugins` to `if/builtins`. This happened with individual PRs for each plugin, and there is some residual cleanup of the IF repository required. This PR covers that cleanup.

**Why**
This ensures we maintain a sensible repo structure and do not break any manifests from our `manifests` folder.

**Context**
This is the final task int he migration of `if-plugins` to `if/builtins`.

**Prerequisites/resources**
All migration PRS must be merged:
#722 
#683 
#681 
#680 
#679 
#678 
#677 
#676 
#675 

plus the new `interpolate` and `exponent` plugins must be migrated (no issues for those yet)

**SoW (scope of work)**

- [ ] split `plugins` and `exhaust` in `builtins` directory
- [ ] audit manifests to ensure `builtins` are imported from the correct directory
- [ ] check all docs to ensure imports and names are correct for all builtins

**Acceptance criteria**

- [ ] All success and failure case manifests in `manifests` import builtins from the correct path

**Given** (Setup): The import paths have been checked
**When** (Action): any manifest from the `manifests` folder is run
**Then** (Assertion): No manifest fails unexpectedly due to an incorrect import path

- [ ] All README documentation includes correct import paths in Typescript and manifest examples
**GIVEN** the import paths have all been checked
**WHEN** I paste the typescript code or manifest content into an IDE and run the code
**THEN** It never fails due to an incorrect import path.

- [ ] The repository structure is as clean and intuitive as possible
**GIVEN** I download `if`
**WHEN** I want to find the source code for a given feature
**THEN** I am able to navigate easily to the right file using the directory names as signposts

**Comment by jmcook1186** [comment]
blocked on merging all migration PRs

**Comment by zanete** [comment]
another ticket is to be created for the tutorial pages in the documentation but this one can be closed down as addressed across multiple other ones
